src/modules/tests/rekey-reason/rekey-reason.selector.ts

```diff
@@ -1,6 +1,7 @@
-import { RekeyReason, StandardCarTestCATBSchema, IpadIssue, Transfer, Other } from '@dvsa/mes-test-schema/categories/B';
+import { RekeyReason, IpadIssue, Transfer, Other } from '@dvsa/mes-test-schema/categories/Common';
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
 
-export const getReasonForRekey = (test: StandardCarTestCATBSchema) => {
+export const getReasonForRekey = (test: CatBUniqueTypes.TestResult) => {
```

Comment:
Think this should be typed to `TestResultSchemasUnion`

Reply:
Changed this